### ISSUE TITLE
1.8.2 cannot handle null as log argument

### ISSUE THREAD

**Issue by objelke** [trentm/node-bunyan#450]
example:

var bunyan = require('bunyan');
var log = bunyan.createLogger({name: 'myapp'});
log.info('hi');
var test = null;
log.info(test);

-->
TypeError: Cannot read property 'err' of null
    at mkRecord (bunyan\lib\bunyan.js:983:48)
    at Logger.info (bunyan\lib\bunyan.js:1042:19)
    at Object.<anonymous> (bunyan_test\app.js:5:5)

up to version 1.8.1 this was not an issue.

**Comment by trentm** [comment]
Guessing this was broken by:

```
- [issue #426] Ensure `log.info({err: err})` results in a "msg" value, just
  like `log.info(err)`.
```

**Comment by trentm** [comment]
#426

**Comment by trentm** [comment]
Thanks! Fixed and published as bunyan@1.8.3
